Assets/Scripts/ScoreMenu.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.UI;
+
+/// <summary>
+/// ScoreMenu is used to display how well the player did in the experience.
+/// </summary>
+public class ScoreMenu 
+    : MonoBehaviour
+{
+    public Text ShotsDisplay;
+    public Text EnemiesDisplay;
+    public Text AccuracyDisplay;
+
+    public void OnValidate()
+    {
+        Assert.IsNotNull(EnemiesDisplay, "EnemiesDisplay should not be null.");
+        Assert.IsNotNull(ShotsDisplay, "ShotsDisplay should not be null.");
+        Assert.IsNotNull(AccuracyDisplay, "AccuracyDisplay should not be null.");
+    }
+
+    public IEnumerator DisplayScoreCoro(int shotsFired, int enemiesHit)
+    {
+        var shotsText = $"Shots Fired: {shotsFired}";
+        var enemiesText = $"Enemies Destroyed: {enemiesHit}";
+        var accuracy = ((float) enemiesHit / shotsFired) * 100;
+        var accText = $"Accuracy: {accuracy:F2}%";
+
+        yield return new WaitForSeconds(0.25f);
```

Comment:
You can add delay in the Coroutine. 

StartCoroutine(WriteTextCoro(ShotsDisplay, shotsText, startIn: 0.25F));
StartCoroutine(WriteTextCoro(ShotsDisplay, shotsText, startIn: 0.5F));
StartCoroutine(WriteTextCoro(ShotsDisplay, shotsText, startIn: 0.75F));

Marked as resolve as it's not a problem and just a comment.